From 51d03dfaf36fd0b4ab399f0fba6ab544a8ed7bec Mon Sep 17 00:00:00 2001
From: Ashley <github@victorianfox.com>
Date: Sat, 16 Sep 2017 11:04:09 +0200
Subject: [PATCH] Add build-arg baseimage to support other archs

---
 Dockerfile | 5 +++--
 README.md  | 4 ++++
 2 files changed, 7 insertions(+), 2 deletions(-)

diff --git a/Dockerfile b/Dockerfile
index ce28ec9..ead1229 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -1,7 +1,8 @@
-FROM node:8.2.1-alpine
+ARG baseimage=node:8.5.0-alpine
+FROM $baseimage

 LABEL com.purrplingcat.name="PurrplingBot"
-LABEL com.purrplingcat.version="1.2.0"
+LABEL com.purrplingcat.version="1.3.0"
 LABEL com.purrplingcat.vendor="PurrplingCat"
 LABEL com.purrplingcat.email="dev@purrplingcat.com"
 LABEL com.purrplingcat.github="https://github.com/EllenFawkes/PurrplingBot"
diff --git a/README.md b/README.md
index 4566d10..8765ef2 100644
--- a/README.md
+++ b/README.md
@@ -66,6 +66,10 @@ docker run -ti purrplingcat/purrplingbot:<version>
 3. Build your Docker image `docker build -t <yourname>/purrplingbot .`
 4. Run your container `docker run -ti <yourname>/purrplingbot`

+### Build your own non-AMD64 image
+
+1. Same as previous, however you need to `docker build --build-arg baseimage=arm64v8/node:8.5.0-slim -t <yourname>/purrlingbot` or whichever is relevant for your architecture.`
+
 ## Bug reporting

 If you find a bug, then report an issue on my github. REMEBER: PurrplingBot is in BETA!
